web/samples_index/web/add_a_drawer_to_a_screen.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```

Comment:
Are these HTML files in /web the generated ones built by the samples index grinder tasks? If so, are they intended to be regenerated each time someone goes to publish the index? And if *that's* so, could they be left out of source control?

Reply:
Yep, it looks like things are still working without them.